.buildbot.sh

```diff
@@ -9,11 +9,26 @@ export RUSTFLAGS="--cfg grmtools_extra_checks"
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup.sh
 sh rustup.sh --default-host x86_64-unknown-linux-gnu --default-toolchain stable -y --no-modify-path
 
-export PATH=`pwd`/.cargo/bin/:$PATH
+# wasm32-wasip2 target and utilities.
+# WASMTIME really wants to modify a file.
+touch .fake_profile
+export WASMTIME_HOME="`pwd`/.wasmtime"
+rustup target add wasm32-wasip2
+PROFILE=.fake_profile curl https://wasmtime.dev/install.sh -sSf | bash
```

Comment:
This one doesn't seem to check whether the current install is sufficient before
downloading/going through the install process, so it redownloads wasmtime every time.

I guess we could just check `which wasmtime` and if we ever need specific versions we can start comparing the output of `wasmtime -v`

Reply:
Kind of a hack, but [99cfb93](https://github.com/softdevteam/grmtools/pull/527/commits/99cfb935968ab1e492685f5d39463f3203ee3c69) checks the PATH for wasmtime before install.

Reply:
If/when an appropriate wasmtime is available on Debian, we can edit the docker file in the repo (which buildbot uses). I guess it's either not present or too old?

Reply:
I feel it is worth trying, there are versions of rust-wasm and nodejs on debian.
I don't know where this docker file is however.

I'm actually not certain which versions are required, but since `wasm-pack` has been around and stable for a long time, I would hope the version of nodejs installed there is sufficient.
I don't know what the necessary wasmtime version is, because I've only ever run with the latest version.

https://packages.debian.org/search?searchon=sourcenames&keywords=rust-wasmtime
https://packages.debian.org/search?searchon=sourcenames&keywords=nodejs

There doesn't appear to be a package for wasm-pack.

Reply:
The docker file is `.buildbot_dockerfile_debian`. You'd need to add the appropriate package(s) to the `apt-get -y install` line. I suggest we give it a go, remove the wasmtime manual installation stuff, and only add that back in if we find we have to. If the version in Debian *is* good enough, it'll save us some long term maintenance (it also makes CI a bit quicker!).

Reply:
Hmm, I don't see a `.buildbot_dockerfile_debian` could it be just not checked into the repo?

Reply:
Ooooops! PR up which adds one.

Reply:
Thanks if it's alright, I'll rebase onto master once that goes through,
There are some other ci changes like the license check on master which will also help.

Reply:
Agreed, if that PR goes in first, it should make this PR easier (I hope).

Reply:
Changed to use debian packages in [c679aec](https://github.com/softdevteam/grmtools/pull/527/commits/c679aec6563cfec6b4386095d4fb8120c4d2e010) and [52be2ee](https://github.com/softdevteam/grmtools/pull/527/commits/52be2ee81a616020d25635413583948b880f059b)